nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -0,0 +1,281 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  inherit (lib) mkEnableOption mkForce mkIf mkMerge mkOption recursiveUpdate types;
+
+  cfg = config.services.dokuwiki;
+  fpm = config.services.phpfpm.pools.dokuwiki;
+
+  user = config.services.nginx.user;
+  group = config.services.nginx.group;
+
+  pkg = pkgs.dokuwiki;
```

Comment:
This is used only once, so probably not worth burning a variable on it.

Reply:
Fixed. Same for the fpm.